dockstore-webservice/src/main/java/io/dockstore/webservice/DockstoreWebserviceConfiguration.java

```diff
@@ -744,6 +744,12 @@ public void setWorkflowVersionLimit(int workflowVersionLimit) {
 
         private String composeSetupVersion;
 
+        private String cwlParsingLambdaVersion;
```

Comment:
Any thoughts on whether the same lambda will handle parsing and packing in the end?

Reply:
Probably not. The LanguageParsingRequest and LanguageParsingResponse doesn't quite match up with cwlpack. It would only be viable if we decide to add a feature where when we register a new workflow version, we immediately pack it.

Reply:
Then it might be worthwhile adding in `cwlPackLambdaVersion` and `wdlPackLambdaVersion`

Reply:
hmm..right now it's part of one of the 6 issues that will be created after story